src/Microsoft.TemplateEngine.Abstractions/IPostAction.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.TemplateEngine.Abstractions
+{
+    public interface IPostAction
+    {
+        int Order { get; }
+
+        string Name { get; }
+
+        IReadOnlyList<IPostActionOperation> Operations { get; }
+
+        IReadOnlyList<IPostActionOperation> AlternateOperations { get; }
```

Comment:
Is this alternate in the send of run one and then another or should this be Alternative operations?